### ISSUE TITLE
!deepFilter ModuleNotFoundError

### ISSUE THREAD

**Issue by youssefabdelm** [Rikorose/DeepFilterNet#208]
When running on Runpod GPU cloud, I get this after install and running `!deepFilter` (it works fine on Colab but somehow for some reason not Runpod):

I installed via `pip install`
```
Traceback (most recent call last):
  File "/opt/conda/bin/deepFilter", line 5, in <module>
    from df.enhance import run
  File "/opt/conda/lib/python3.7/site-packages/df/enhance.py", line 14, in <module>
    from df.io import load_audio, resample, save_audio
ModuleNotFoundError: No module named 'df.io'
```

**Comment by Rikorose** [comment]
Which version did you install?

**Comment by Rikorose** [comment]
This is fixed in v0.3.1. You need to update.

**Comment by youssefabdelm** [comment]
> FYI an upcoming release will have a slightly improved noise reduction via the deep-filter executable

Can't wait! I'll try `deep-filter` too.
